bin/run_ascat.r

```diff
@@ -1,31 +1,58 @@
-#!/bin/env Rscript
-args = commandArgs(trailingOnly=TRUE)
-if(length(args)<6){
-    stop("No input files supplied\n\nUsage:\nRscript run_ascat.r tumor_baf tumor_logr normal_baf normal_logr tumor_sample_name baseDir gcfile\n\n")
-} else{
-    tumorbaf = args[1]
-    tumorlogr = args[2]
-    normalbaf = args[3]
-    normallogr = args[4]
-    tumorname = args[5]
-    baseDir = args[6]
-    gcfile = args[7]
-}
+#!/usr/bin/env Rscript
+library("optparse")
```

Comment:
I was guessing yes, because I had people using that on offline clusters.
But I had to check, so it is indeed in the environment, but only because `QualiMap` has it in its [recipe](https://github.com/bioconda/bioconda-recipes/blob/e0ea1ecf1c842c7ea2f681485b1063010c5247d1/recipes/qualimap/meta.yaml#L18).
Thanks for asking about it, I will need to remember that when we want to go to DSL2, which should now be sooner :-D